packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,942 @@
+
+Don’t prevent form submission by disabling the submit button. An error message can give more information than a disabled button can to help users figure out which fields are invalid.
+
+### Errors
+
+Disable or remove any options in a select that would result in an error if possible. If an error occurs, use inline error text to explain how to fix an error.
+
+Ideally, help text should have enough information to help users prevent errors. If help text exists and you need to show an error, the error text should replace and repeat the help text until the error has been resolved.
+
+Error text should:
+
+<UnorderedList>
+  <ListItem>
+    Be actionable. Explain how to fix an error and if reasonable, why it happened so that it might also be prevented in
+    the future.
+  </ListItem>
+  <ListItem>
+    Be concise and simple, maybe even more than normal. Avoid jargon. Try to keep error text to 2 sentences or fewer.
+  </ListItem>
+  <ListItem>
+    Use the passive voice for input errors to avoid placing blame on the user. For example, “A friendly name is
+    required.”
+  </ListItem>
+  <ListItem>
+    Use the active voice for system errors. For example, “Our systems are currently down. Please contact our support
+    team.”
+  </ListItem>
+</UnorderedList>
+
+### Optional compositional elements
+
+- **Prefix/suffix** — Use a prefix or suffix to help users format their input and to provide more context about the value a user is entering. For example, you could prefix or suffix a field with a currency symbol, or use suffix to append a character counter. Make sure to consider internationalization when using prefixes or suffixes since formatting may differ across languages. For example, currency symbols are placed on the left in American English, while they're placed on the right in French. Don't use prefix/suffix text as a replacement for a label.
+- **Icon** — Use an icon if you need to give the user additional controls for the field. For example, use an icon to clear a field on click, removing the need for users to manually delete their entered value. Place icon buttons that trigger an action on the right side of the field. If you need 2 actions on a field (e.g., popover trigger and clear field), place the icon button that affects the field value on the right, and the other icon on the left.
+
+## When to use a Select
+
+Use a select when:
+
+<UnorderedList>
+  <ListItem>You need a native picker experience, especially on mobile devices.</ListItem>
+  <ListItem>You need the component to work out-of-the-box across all operating systems and browsers.</ListItem>
+</UnorderedList>
+
+<DoDont>
+  <Do center title="Do" body="Sort options in a list logically (*e.g.*, alphabetically, ascending, descending).">
```

Comment:
Seems like "e.g." isn't getting the italic style since it's in a component.

Reply:
@serifluous I updated all `e.g.` --> _`e.g.`_, except for the one that occurs in the do/don't example since the body property does not accept a react node (only unformatted text). Want to confirm that is the intended styling, if not, should I remove all italicized _`e.g.`_?

Reply:
nice, thanks for doing that! yep, they should all be italicized, but if we can't do it in do/don't only, that's fine.